config/breadcrumbs.rb

```diff
@@ -12,6 +12,10 @@
   parent :public_node, Node.root_node
 end
 
+crumb :public_root do
+  link 'Home', root_path
+end
+
```

Comment:
We don't have a `:public_root` crumb anymore. We have `:public_node, Node.root_node` instead.